### ISSUE TITLE
Forecast File Inputs not copying?

### ISSUE THREAD

**Issue by Mkrig24** [nmfs-ost/SSMSE#214]
I am running an MSE using sablefish stock assessment files, which have catch inputs specified at the bottom of the forecast file that cap catch at the ACLs set by council in the first two years of the model. When the model runs outside of SSMSE these inputs correctly cap the catch, but I noticed that inside of run_SSMSE, when the forecast file gets copied to _EM_init these input values aren't copied over, and the catch ends up being significantly higher. 

![image](https://github.com/nmfs-fish-tools/SSMSE/assets/31481992/042851ef-a62a-43a4-bfd7-4d77018cc1f5)
becomes 
![image](https://github.com/nmfs-fish-tools/SSMSE/assets/31481992/cc27781c-1ef1-46c3-9235-d0b8ae25ed42)

Is there a way to make sure these inputs are copied over? Or another work around to get catch specified correctly in these first two years?

**Comment by k-doering-NOAA** [comment]
@Mkrig24 thank you for the question! Here is the code that is removing the forecast catch inputs: https://github.com/nmfs-fish-tools/SSMSE/blob/1f12b27f65403062d914a79d6e239e5ccb3e9dc0/R/manipulate_EM.R#L395-L402

This was done because there isn't one obvious way to apply forecast catch inputs as the simulation projects forward in time without the user providing additional information.

Right now, I believe the only way to get catch specified in the forecast file would be to [develop a custom management procedure](https://nmfs-fish-tools.github.io/SSMSE/manual/SSMSE.html#custom-MS) - @nathanvaughan-NOAA, does that sound correct?

**Comment by nathanvaughan-NOAA** [comment]
That is all correct @k-doering-NOAA we have similar issues in the Southeast with a lag between the assessment data end year and management implementation year but don't have a good solution at the moment for defaulting this setup in SSMSE. A custom management procedure is likely the only solution if you want to explicitly include this in an MSE. @Mkrig24 if you look at the merge_catch_bias branch we are currently working on developing some custom EM's in the ratioBiasEM.R file that may help you with figuring out how to setup your own custom EM.

**Comment by Mkrig24** [comment]
Thanks! I looked at the ratioBiasEM.R as a place to start @nathanvaughan-NOAA , and created a similar file that runs as expected as a custom EM. However, when I change from from run_parallel = FALSE to TRUE, the iterations fail before the init_dat.ss is copied over into the EM_init folder. I'm confused because these calls work as expected when parallelization is off. Also, if I run the standard MS "EM" instead of my custom EM, run_parallel = TRUE runs successfully. Any ideas on why or where to look to debug this? Have you tested your custom EM running in parallel?

**Comment by k-doering-NOAA** [comment]
closing, as I believe this has been answered, but it can be reopened if not!
